php82-moodle-unit/Dockerfile

```diff
@@ -81,8 +83,20 @@ COPY conf.json /var/lib/unit/
 
 COPY xx-moodle.ini /etc/php82/conf.d/xx-moodle.ini
 
+# S6 & cron management
+ADD https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-noarch.tar.xz /tmp
+RUN tar -C / -Jxpf /tmp/s6-overlay-noarch.tar.xz
+ADD https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-x86_64.tar.xz /tmp
```

Comment:
Curious why you're downloading it instead of installing via `apk add s6`

Reply:
Yes, it comes from official s6-overlay documentation, but actually not need as s6-overlay package exists for alpine. Removed in last commit